recipes/fast-dp/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "fast-dp" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/fast_dp/fast_dp-{{ version }}.tar.gz
+  sha256: f7cd04ede50759dc80840af51b12739e78edad32bc4b55e9dab06e8c920f91ed
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - fast_dp = fast_dp.fast_dp:main
+    - fast_rdp = fast_dp.fast_rdp:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - fast_dp
+
```

Comment:
You should probable test for the commands/entry_points in `test` as well, e.g.:
```
  commands:
    - fast_dp
    - fast_rdp
```

Reply:
Unfortunately even using just the `--help` parameter requires cctbx (one of the external dependencies) to be present, so, for now, this will fail in the build step.

There is work ongoing to create a conda-forge package for cctbx (#10021), but that will be some way away.